File: src/main/java/com/thoo/api/model/AESModel.java
package com.thoo.api.model;

import java.util.Date;

public final class AESModel {

    public String build;
    public String mainKey;
    public DynamicPak[] dynamicPaks;
    public Date updated;

    public final static class DynamicPak {
        public String pakFilename;
        public String pakGuid;
        public String key;
    }

}

File: src/main/java/com/thoo/api/models/AESModel.kt
package com.thoo.api.models

import java.util.*

class AESModel(
        val build: String,
        val mainKey: String,
        val dynamicKeys: Array<DynamicPak>,
        val updated: Date
)

class DynamicPak(
        val pakFilename: String,
        val pakGuid: String,
        val key: String
)